ports/stm32/flash.c

```diff
@@ -71,7 +71,7 @@ static const flash_layout_t flash_layout[] = {
 #elif defined(STM32H7)
 
 static const flash_layout_t flash_layout[] = {
-    { 0x08000000, 0x20000, 8 },
+    { 0x08000000, 0x20000, 16 },
```

Comment:
I guess this is right... @iabdalkader why did you have it as 8 in your original code change?

Reply:
It should be 16 yes.